executor/compiler.go

```diff
@@ -247,6 +247,41 @@ func getStmtDbLabel(stmtNode ast.StmtNode) map[string]struct{} {
 			dbLabel := x.Table.Schema.O
 			dbLabelSet[dbLabel] = struct{}{}
 		}
+	case *ast.TruncateTableStmt:
```

Comment:
We have a lot of duplicate codes now...IMHO it's better to refine the definition of those `TruncateTableStatement`, `RepairTableStmt`...

Well again, we don't have to make it in this PR.

Reply:
I'm sorry about the duplicate code, but I don't have a good solution to deal with different types of stmt getting their dbname :(   Do you have any good suggestions?